tools/cockpit.spec

```diff
@@ -282,8 +282,6 @@ done
 rm -r %{buildroot}/%{_prefix}/%{__lib}/cockpit-test-assets
 # files from -pcp
 rm -r %{buildroot}/%{_libexecdir}/cockpit-pcp %{buildroot}/%{_localstatedir}/lib/pcp/
-# files from -machines
```

Comment:
There are more instances of "machines" in the file, such as in the comment in line 66, and building machines.list (which is the command that causes the rpm build failure).

Reply:
Thanks for the hint!